R/step-subset.R

```diff
@@ -135,6 +135,47 @@ select.dtplyr_step <- function(.data, ...) {
   step_group(out, groups)
 }
 
+#' @importFrom dplyr relocate
```

Comment:
I think we should probably just bump the dplyr dependency in the DESCRIPTION to 1.0.0